### PR TITLE
Add inner zoom to the product's image

### PR DESCRIPTION
**Pull request by vitoriaHeliane**
#### What is the purpose of this pull request?
Add inner zoom to the product's selected image.

#### What problem is this solving?
Add this feature.

#### How should this be manually tested?
https://waza--storecomponents.myvtex.com

#### Types of changes
- [ ] Bug fix (a non-breaking change which fixes an issue)
- [X] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -1,24 +0,0 @@
-{
```
Comment (andreldsa) [inline .eslintrc]: Revert

File: react/components/ProductImages/components/SelectedImage.js [hunk 2]
```diff
@@ -1,23 +1,47 @@
...
 
 import VTEXClasses from '../constants/productImagesClasses'
 
 /**
  * Selected Image component.
  * Display an image.
  */
-class SelectedImage extends Component {
+export default class SelectedImage extends Component {
+  constructor(props) {
```
Comment (salesfelipe) [inline react/components/ProductImages/components/SelectedImage.js]: No need to user the constructor, you can change it for: 

state = {
    showZoom: false
}

File: react/components/ProductImages/components/InnerZoom.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+            style={zoomStyle}>
+            {children}
+          </div>
+        </div>
+      </div>
+    )
+  }
+}
+
+InnerZoom.propTypes = {
```
Comment (brunojdo) [inline react/components/ProductImages/components/InnerZoom.js]: Put this propTypes into class.

Comment (vitoriaHeliane) [reply react/components/ProductImages/components/InnerZoom.js]: DOne!

File: react/components/ProductImages/components/SelectedImage.js (hunk 2 above)
Comment (vitoriaHeliane) [reply react/components/ProductImages/components/SelectedImage.js]: Done!

**Pull request merged**